pkg/detector/library/driver.go

```diff
@@ -57,17 +51,17 @@ type Driver struct {
 	advisories []advisory
 }
 
-// NewDriver is the factory method from drier
-func NewDriver(advisories ...advisory) Driver {
+// Aggregate aggregates drivers
+func Aggregate(advisories ...advisory) Driver {
```

Comment:
Aggregate return driver, but the driver has not ecosystem. 
driver.Type() method return empty.